src/helix-ui/styles/components/choice-tiles.less

```diff
@@ -67,7 +67,7 @@ label.hxChoice > input:checked + hx-tile {
   border-color: @cyan-900;
   color: @cyan-900;
 
-  hx-tile-description {
```

Comment:
Make sure to move/copy the old definitions to a section at the bottom of this file for deprecation.